### ISSUE TITLE
metro 0.25.1 can not be used from npm

### ISSUE THREAD

**Issue by vovkasm** [facebook/metro#142]
**What is the current behavior?**
Can't install metro@0.25.1 or have it as dependency.

**If the current behavior is a bug, please provide the steps to reproduce and a minimal repository on GitHub that we can `yarn install` and `yarn test`.**
```sh
mkdir fuck-metro && cd fuck-metro
npm init # all answers default
npm i -P metro@0.25.1 
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "npm:": npm:babylon@^7.0.0-beta
```

**What is the expected behavior?**
Package `metro` should successfully be installed into `node_modules` and recorded in `package.json`

**Please provide your exact Metro configuration and mention your Metro, node, yarn/npm version and operating system.**
OS: macOS Sierra 10.12.6 (16G1212)
```
npm --versions
{ 'fuck-metro': '1.0.0',
  npm: '5.6.0',
  ares: '1.13.0',
  cldr: '32.0.1',
  http_parser: '2.7.0',
  icu: '60.2',
  modules: '59',
  napi: '2',
  nghttp2: '1.29.0',
  node: '9.5.0',
  openssl: '1.0.2n',
  tz: '2017c',
  unicode: '10.0',
  uv: '1.19.1',
  v8: '6.2.414.46-node.18',
  zlib: '1.2.11' }
```

Error exactly here: [packages/metro/package.json](https://github.com/facebook/metro/blob/v0.25.1/packages/metro/package.json#L56) and it still in master.

And in case one can't find it, I provide link to documentation of `package.json` format: https://docs.npmjs.com/files/package.json#urls-as-dependencies

**Comment by koloff** [comment]
+1

**Comment by vovkasm** [comment]
@davidaurelio As a reviewer of breaking change, can you comment on this? Thanks!

**Comment by vovkasm** [comment]
@qfox does commit https://github.com/facebook/metro/commit/9ac44657bcae49a6fb82d603a0120338653a7558 is your?

**Comment by jeanlauliac** [comment]
A way to fix this, I think, is to create a published package ex. `metro-babylon7` that itself depends on `babylon@7` and re-export it verbatim, then depend on `babylon@6` and `metro-babylon7` from `metro`.

**Comment by vovkasm** [comment]
@jeanlauliac Looks like solution, should I create pull request?

**Comment by vovkasm** [comment]
Thanks all! Fix landed in master and will be in 0.28 release (I hope). So I close the issue.
